client/src/img/ui/dot.svg

```diff
@@ -61,7 +61,7 @@
      id="defs1547" /><g
      transform="translate(1,14)"
      id="g3"
-     style="fill:#ffffff;opacity:0.5"><g
+     style="fill:#ffffff;opacity:0.2"><g
```

Comment:
Here are dots with 20% opacity in Firefox and Chrome. Are you sure you want this opacity? In this case we better remove dots, because they're almost invisible.
![image](https://user-images.githubusercontent.com/6031763/50044634-a972c700-0086-11e9-9397-daf19ea10d0c.png)

Reply:
Then we can change style -  r="0.5" to  r="1"

Reply:
Can You please create separate PR for this?